devenv/README.md

```diff
@@ -142,10 +142,10 @@ cd /devenv/sbtc
 After the deployment is up, generate a new private key:
 
 ```
-./devenv/sbtc/bin/sbtc generate-from new
+./devenv/sbtc/bin/sbtc generate-from -b regtest -s testnet new
 ```
 
-Take the WIF + mnemonic phrase and update the sbtc config:
+Take the mnemonic phrase and update the sbtc config:
```

Comment:
Does it actually not need the WIF? or does it need it and the default config file is wrong?

Reply:
Kk no wif needed. Then LGTM